src/wallet/api/stake_unlock_result.h

```diff
@@ -0,0 +1,26 @@
+#include "wallet/api/wallet2_api.h"
+#include "wallet/wallet2.h"
+
+#include <string>
+
+
+namespace Wallet {
+
+class WalletImpl;
+class StakeUnlockResultImpl : public StakeUnlockResult
+{
+public:
+    StakeUnlockResultImpl(tools::wallet2::request_stake_unlock_result res);
+    StakeUnlockResultImpl();
+    ~StakeUnlockResultImpl();
+
+    bool success();
+    std::string msg();
+    PendingTransaction * ptx();
```

Comment:
These should be marked `override`, i.e. `bool success() override;`.  It won't change anything about how they work, but it makes it explicit that they are overriding a base class virtual method (and if such a base class method doesn't exist or isn't virtual, compilation will fail).

Reply:
Also the spaces on *both* sides of the `*` hurt my eyes (I wrote more on this below).